Cognigy/WebchatWidget
# Feature/55717 add rating widget

This PR adds the Chat options screen.

How to test:

1. Run the webchat.
2. Open the Conversation and click the secondary button to the left of the close button
3. This will open the chat options screen
4. You should be able to submit a feedback. Once the feed is submitted, you see an event banner at the top
5. You should also be able to click the postback buttons. Check if the texts are posted back to the conversation
